File: scripts/analyze_bundle.pl
#!/usr/bin/perl
use strict;
use warnings;
my $data = do { local $/; open my $fh, "<", "/home/drarmank/public_html/assets/index-DJeWhCy-.js" or die $!; <$fh> };
print "size: ", length($data), "\n";
my @markers = ("upsertPatient","createPatient","medicare_patients","phpAuthToken","sync.php",".php","fetch(","localStorage","medicare_sync_queue","canister","actor","apiClient","patients/create","/api/patients","registerPatient","savePatient");
for my $m (@markers) {
    my $c = () = $data =~ /\Q$m\E/g;
    print "$m: $c\n";
}
print "\n=== first upsertPatient ctx ===\n";
my $pos = index($data, "upsertPatient");
if ($pos >= ) { my $s = $pos-300 <  ?  : $pos-300; print substr($data, $s, 800), "\n"; }
print "\n=== first createPatient ctx ===\n";
$pos = index($data, "createPatient");
if ($pos >= ) { my $s = $pos-300 <  ?  : $pos-300; print substr($data, $s, 800), "\n"; }
print "\n=== first medicare_patients ctx ===\n";
$pos = index($data, "medicare_patients");
if ($pos >= ) { my $s = $pos-200 <  ?  : $pos-200; print substr($data, $s, 600), "\n"; }
print "\n=== phpAuthToken ctx ===\n";
$pos = index($data, "phpAuthToken");
if ($pos >= ) { my $s = $pos-200 <  ?  : $pos-200; print substr($data, $s, 500), "\n"; } else { print "NOT FOUND\n"; }

File: tmp/analyze_bundle.php
<?php
$data = file_get_contents('/home/drarmank/public_html/assets/index-DJeWhCy-.js');
echo "size: " . strlen($data) . "\n";
$markers = ['upsertPatient', 'createPatient', 'medicare_patients', 'phpAuthToken', 'sync.php', '.php', 'fetch(', 'localStorage', 'medicare_sync_queue', 'canister', 'actor', 'apiClient', 'patients/create', '/api/patients', 'registerPatient', 'savePatient'];
foreach ($markers as $m) {
    echo $m . ": " . substr_count($data, $m) . "\n";
}
echo "\n=== first upsertPatient ctx ===\n";
$pos = strpos($data, 'upsertPatient');
if ($pos !== false) {
    echo substr($data, max(, $pos-300), 800) . "\n";
}
echo "\n=== first createPatient ctx ===\n";
$pos = strpos($data, 'createPatient');
if ($pos !== false) {
    echo substr($data, max(, $pos-300), 800) . "\n";
}
echo "\n=== first medicare_patients ctx ===\n";
$pos = strpos($data, 'medicare_patients');
if ($pos !== false) {
    echo substr($data, max(, $pos-200), 600) . "\n";
}
echo "\n=== phpAuthToken ctx ===\n";
$pos = strpos($data, 'phpAuthToken');
if ($pos !== false) {
    echo substr($data, max(, $pos-200), 500) . "\n";
} else {
    echo "NOT FOUND\n";
}
